hal/src/samd51/sercom/pads.rs

```diff
@@ -240,37 +240,42 @@ pad!(Sercom5Pad3 {
 // sercom6[2]:  PC06:C   PC10:C   PC14:D   PC18:C   PD10:D
 // sercom6[3]:  PC07:C   PC11:C   PC15:D   PC19:C   PD11:D
 
-#[cfg(feature = "samd51p19a")]
+#[cfg(any(feature = "samd51p19a", feature = "samd51n20a"))]
```

Comment:
I think this is the correct way to handle this. Both N and P variants have the extra SERCOMs, but the N variant is missing some pins. Some of these pins should only apply when targeting a P variant. Will the attributes within the `pad!` macro override the one before the `pad!` macro?

Reply:
This is the correct approach, yes.

The top-level `#[cfg()]` attributes (that which you've quoted) will determine whether or not to invoke the `pad!` macro for the given `Sercom`/`Pad` combination. Then, within the `pad!` invocation, additional `#[cfg()]` attributes will determine whether or not to include that specific pin.